Project 'tests' not found

Hello,
I want to test the install with command

`./gradlew :tests:testSystemBasic -Dwhisk.auth=$WHISK_AUTH -Dwhisk.server=https://$WHISK_SERVER -Dopenwhisk.home=`pwd` --info
`

as mentioned in Readme
but the tests directors seems missing????
I have output:

`
> Configure project : 
Evaluating root project incubator-openwhisk-deploy-kube using build file /home/xxx/incubator-openwhisk-deploy-kube/build.gradle
All projects evaluated.
FAILURE: Build failed with an exception.
What went wrong:
Project 'tests' not found in root project 'incubator-openwhisk-deploy-kube'.
`

--- a/README.md
+++ b/README.md
@@ -350,7 +350,9 @@ etc.) are not directly accessible from the outside of the Kubernetes cluster.
 Therefore, although you can run full system tests against a
 Kubernetes-based deployment by giving some extra command line
 arguments, any unit tests that assume direct access to one of the internal
-microservices will fail.   The system tests can be executed in a
+microservices will fail.   First clone the [core OpenWhisk repository](https://github.com/apache/openwhisk)
+locally and set `$OPENWHISK_HOME` to its top-level directory. Then, the
+system tests can be executed in a
 batch-style as shown below, where WHISK_SERVER and WHISK_AUTH are
 replaced by the values returned by `wsk property get --apihost` and
 `wsk property get --auth` respectively.

